enableDetective.py

```diff
@@ -228,7 +232,7 @@ def _master_memberList(g: str) -> typing.List[typing.Dict]:
             {g: {x['AccountId'] for x in v if x['Status'] == 'INVITED'} for g, v in pending})
 
 
-def create_members(d_client: botocore.client.BaseClient, graph_arn: str, account_ids: typing.Set[str],
+def create_members(d_client: botocore.client.BaseClient, graph_arn: str, disable_email: bool, account_ids: typing.Set[str],
```

Comment:
I guess it would be changing the contract for the set of account ids to make it an optional arg?  What happens if someone runs the old version of this library against new AM (I assume it defaults to disabled = false)?

Reply:
Oh hm, is this method signature considered part of the library interface? I was thinking the CLI was the only stable interface (where it defaults to false), but if people are scripting directly against this method, this will break them, and I should move it last and make it optional.

@ruslanafr, have these methods been considered part of the interface? (If so, we probably need to get docs on them)